taskchampion/xtask/src/main.rs

```diff
@@ -3,17 +3,33 @@
 //! At the moment it is very simple, but if this grows more subcommands then
 //! it will be sensible to use `clap` or another similar library.
 
+//! const MSRV_FILE_PATHS in /xtask/main.rs is a list of relative file paths that the
+//! Minimum Supported Rust Version should be commented into
+
+/// Xtask Auto-Generated Comments:
+///  Minimum Supported Rust Version for this crate:
+///  MSRV = "1.63"
 use std::env;
 use std::fs::File;
 use std::io::Write;
-use std::path::PathBuf;
+use std::io::{BufRead, BufReader};
+use std::os::unix::prelude::FileExt;
+use std::path::{Path, PathBuf};
+
+// Increment length of array when adding more paths.
+const MSRV_FILE_PATHS: [&str; 2] = [r"main.rs", r"src/main.rs"];
```

Comment:
I can't quite tell where these paths are relative to?

Cargo sets `$CARGO_MANIFEST_DIR` to the `taskchampion/xtask` directory, so you might be able to build paths relative to there, and then use that environment variable to make `xtask msrv` independent of the current directory.

Reply:
Oh, I'm sorry. These are left over test paths. One fails, one succeeds.